sources

```diff
@@ -0,0 +1,2 @@
+401aa1438604b8e32306fa2bd4f8e211  node-v8.9.4-headers.tar.gz
```

Comment:
Curious what this file is for, its a tad cryptic.

Reply:
Yeah; it's for identifying sources downstream in dist-git; and unfortunately the mechanism breaks if I put any comments in the file 🙄

Arguably this doesn't belong in this upstream repo. But try as we might, the line between where ownership belongs on translating upstream into production builds remains fuzzy, mostly erring on the side of leaving control to the image owner. In this particular case this will likely be revised as we try to automate the process of updating bundled sources.